tests/integration/test_monitors.py

```diff
@@ -10,6 +11,26 @@ def generate_check_in(slug):
     }
 
 
+def test_monitor_ingest(mini_sentry, relay):
```

Comment:
Not sure if I understand. What's the behavior we're trying to test? Looking at the test below the only difference I see is the `monitors_consumer` fixture, but not relay behavior. Maybe I'm missing something?

Reply:
This test doesn’t run the full processing pipeline. I believe it’s faster cc @jan-auer

Reply:
That's right. This test runs without processing mode and doesn't need Kafka, so it's a fair bit faster to run.